<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1550</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1550</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>İstenmeyen Nesne Tespiti Uygulaması</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>741</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>Kaynak</string>
    </property>
    <widget class="QLabel" name="kaynak">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>720</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>720</width>
       <height>600</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>781</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>İşlenmiş</string>
    </property>
    <widget class="QLabel" name="sonuc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>720</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>720</width>
       <height>600</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>961</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Ayarlar</string>
    </property>
    <widget class="QComboBox" name="model_type">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Yolo Model:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kaynak Dosya:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="metin_kaynak">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="b_kaynak_ac">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aç</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Model Size:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="model_size">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hedef Dosya:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_hedef_ac">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aç</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="metin_hedef">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cuda_kullan">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>80</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CUDA Destekli Hesapla</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>630</y>
      <width>561</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>İşleme</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>541</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>190</y>
       <width>502</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Current Frame:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="curr_frame_counter">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total Frame:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="total_frame_counter">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb2">
        <property name="text">
         <string>Başla</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1550</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionResim_A">
   <property name="text">
    <string>Resim Aç</string>
   </property>
  </action>
  <action name="actionVideo_A">
   <property name="text">
    <string>Video Aç</string>
   </property>
  </action>
  <action name="actionKaydet">
   <property name="text">
    <string>Kaydet</string>
   </property>
  </action>
  <action name="actionDosyay_Kapat">
   <property name="text">
    <string>Dosyayı Kapat</string>
   </property>
  </action>
  <action name="actionProgramdan_k">
   <property name="text">
    <string>Programdan Çık</string>
   </property>
  </action>
  <action name="actionVeri_D_zenleme">
   <property name="text">
    <string>Veri Düzenleme</string>
   </property>
  </action>
  <action name="actionModel_E_itimi">
   <property name="text">
    <string>Model Eğitimi</string>
   </property>
  </action>
  <action name="actionYard_m">
   <property name="text">
    <string>Yardım</string>
   </property>
  </action>
  <action name="actionHakk_nda">
   <property name="text">
    <string>Hakkında</string>
   </property>
  </action>
  <action name="actionAyarlar">
   <property name="text">
    <string>Ayarlar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
